docs/docs/reference/changed/compiler-plugins.md

```diff
@@ -0,0 +1,133 @@
+---
+layout: doc-page
+title: "Changes in Compiler Plugins"
+---
+
+Compiler plugins are supported by Dotty since 0.9. Compared to Scalac, there are
+two notable changes:
+
+- No more support for analyzer plugins
+- Added support for research plugins
+
+[Analyzer plugins][1] in Scalac are executed during type checking to change the
+normal type checking. This is a nice feature for doing research, but for
+production usage, a predictable and consistent type checker is more important.
+
+For experiments and researches that rely on analyzer plugins in Scalac,
+_research plugin_ can be used for the same purpose in Dotty. Research plugins
+are more powerful than Scalac analyzers as it enables plugin authors to
+customize the whole compiler pipeline. That means, you can easily use your
+customized typer to replace the standard typer, or roll your own parser for
+your domain-specific language. Research plugins are only enabled for nightly or
+snaphot releases of Dotty.
+
+The common plugins that add new phases to the compiler pipeline are called
+_standard plugins_ in Dotty. In terms of features, they are similar to
+Scalac plugins, despite minor changes in the API.
+
+## Physical Interface
+
+Both research plugins and standard plugins share the same command line options
+as Scalac plugins. You may manually specify a plugin as a compiler option as follows:
+
+```shell
+dotc -Xplugin:pluginA.jar -Xplugin:pluginB.jar Test.scala
+```
+
+The compiler will examine the jar provided, and look for a property file
+`plugin.properties` in the root directory of the jar. The property file
+specifies the fully qualified plugin class name. The format of a property file
+looks like the following:
+
+```
+pluginClass=dividezero.DivideZero
+```
+
+The above is a change from Scalac, which depends on an XML file
+`scalac-plugin.xml`. Starting from 1.1.5, SBT also supports Dotty compiler plugins:
+
+```Scala
+addCompilerPlugin("org.divbyzero" % "divbyzero" % "1.0")
+```
+
+## Standard Plugin
```

Comment:
Maybe call this "Writing your first plugin", and move the content of `plugin.properties` here?

Reply:
That can be more friendly, but I feel the tutorial-like style doesn't fit the documentation style of other pages.